scala/README.md

```diff
@@ -406,20 +485,12 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
     val good = geo(latitude = a, longitude = b)
     ```
 
-0. Whenever possible, simplify pattern matching expressions by omitting the `match` keyword and using partial functions:
+0. Always pass named `Boolean` parameters to functions, even when they are the only parameter:
```

Comment:
i prefer to rewrite public methods that require a boolean to have two versions:

```scala
class Bad {
  def bad(shouldDoSomething: Boolean)
}

class Good {
  def goodDoSomething() 
  def goodDontDoSomething()

  private def good(doSomething: Boolean)
}
```
(the two good ones just make a call to private method `good` and pass the appropriate boolean)

who's with me